### DISCUSSION TITLE
How can I send a message by replying to a message?

### DISCUSSION THREAD

**Discussion by niyazialpay** [botman/botman#1414] [Q&A]
Hello,

I need to reply to the messages I've sent using $bot->reply(). You can check the screenshot as an example. I couldn't figure out what I need to do to use it this way.

![image](https://github.com/user-attachments/assets/10cec69e-e549-4d30-98d2-9f708f9f630d)

**Comment by filippotoso** [comment]
If you are using the Telegram driver, here you can find the documentation about its APIs: https://core.telegram.org/bots/api

**Reply by niyazialpay** [reply]
How can I get the ID of a message sent with $bot->reply in BotMan? Doesn't this package have such a feature?

**Comment by filippotoso** [comment]
You just need to read the source code to know how...

public function reply($message, $additionalParameters = [])
https://github.com/botman/botman/blob/2.0/src/BotMan.php#L653

public function sendPayload($payload)
https://github.com/botman/botman/blob/2.0/src/BotMan.php#L664

public function sendPayload($payload)
https://github.com/botman/driver-telegram/blob/master/src/TelegramDriver.php#L503C21-L503C32
